packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -0,0 +1,57 @@
+@import '../../mixins/links.css';
+:host {
+  display: block;
+}
+a[aria-expanded='true'] ~ #info {
+  padding-left: calc(2rem - 7px);
+  border-left: 7px solid transparent;
+  border-left-color: var(--color-primary-alt-light);
+}
```

Comment:
Applying the left border to the expanding text only is a bug fix for: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/577